src/Integration/TeamExplorer/ConnectSectionView.xaml

```diff
@@ -90,16 +89,41 @@
                       Margin="6,3,6,6"
                       ItemContainerStyle="{StaticResource SQServerTreeViewItemStyle}"
                       ItemsSource="{Binding State.ConnectedServers}"
-                      Style="{StaticResource SQTreeViewStyle}">
+                      Style="{StaticResource SQTreeViewStyle}"
+                      VirtualizingPanel.IsVirtualizing="True">
```

Comment:
I think IsVirtualizing=true is not needed because the TreeView is in a vertical StackPanel, which will break the virtualization anyway (because it measures its children with Infinity). May be you should remove this line to avoid confusion in the future.